openmdao/core/component.py

```diff
@@ -1580,5 +1629,5 @@ def items(self):
         return [(key, self._dict[key]['value']) for key in self._dict]
 
     def iteritems(self):
-        for key, val in self._dict.iteritems():
+        for key, val in self._dict.items():
```

Comment:
How did we miss this? Must not be covered.

Reply:
Turns out iteritems was being referenced from some dead code so I removed it.